src/Graph.elm

```
@@ -1245,12 +1204,13 @@ heightLevels (AcyclicGraph graph _) =
                     in
                     case go currentLevel1 nextLevel1 indegrees1 (remove source.node.id graph) of
                         [] ->
-                            Debug.crash "Graph.heightLevels: Reached a branch which is impossible by invariants. Please file a bug report!"
+                            -- Debug.crash "Graph.heightLevels: Reached a branch which is impossible by invariants. Please file a bug report!"
```

This will silently fail

---

see above